spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/JdbcChannelMessageStore.java

```diff
@@ -255,6 +259,19 @@ public void setMessageRowMapper(MessageRowMapper messageRowMapper) {
 	}
 
 	/**
+	 * Allows for passing in a custom {@link MessageGroupPreparedStatementSetter}.
+	 * The {@link MessageGroupPreparedStatementSetter} is used to insert message in the database.
+	 *
+	 * @param messageGroupPreparedStatementSetter Must not be null
```

Comment:
The param must have some description - `not null` is not enough.
Also `@since 5.0`